src/components/LandingPage.js

```diff
@@ -2,7 +2,14 @@
 import { jsx } from 'theme-ui';
```

Comment:
In order for theme-ui to work properly, you have to include a comment on the first line right above the import:
```
/** @jsx jsx */
import { jsx } from 'theme-ui';
```